### ISSUE TITLE
The finetuning results on Bridge are not good.

### ISSUE THREAD

**Issue by swstbecrpgmail** [OpenDriveLab/UniVLA#19]
Thank you very much for open-sourcing this outstanding work!
I'm trying to finetune openvla on bridge_oxe hower

I'm trying to finetune OpenVLA on the Bridge-OXE dataset, but my loss curve fluctuates violently, and the model isn't fitting well. Are there any methods to improve the fine-tuning results?

Here's my loss curve(smoothing=0.99), action fitting plots and training parameters.

![Image](https://github.com/user-attachments/assets/91a6461a-ce43-4c38-b8b3-b821f9bf71f2)

![Image](https://github.com/user-attachments/assets/d411365c-6bf3-4b84-9a03-3947264d9ccd)

![Image](https://github.com/user-attachments/assets/323e36ef-2811-4499-b390-2e8a8b882502)

Additionally, I compute the ground-truth actions following bridge_oxe_dataset_transform() function.

![Image](https://github.com/user-attachments/assets/35c7398f-a091-49db-92b8-3129faac6324)

If anyone has any possible suggestions for improvement, such as adjusting parameters or pointing out potential mistakes, feel free to leave comments! Thank you very much!

**Comment by retsuh-bqw** [comment]
Hi! It looks like you're using a small batch size with a relatively high learning rate (in our experiments, we used a batch size of 128 and learning rate of 1e-4). You might want to try a much lower learning rate, such as 1e-5. Also, the L1 loss should typically converge to a value below 0.1.
